ruby/emit_log.rb

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env ruby
-# encoding: utf-8
 
-require "bunny"
+require 'bunny'
 
-conn = Bunny.new(:automatically_recover => false)
-conn.start
+connection = Bunny.new
```

Comment:
Is this change really an improvement? Isn't it obvious what `conn` and `ch` and `x` mean in this small script?

Sorry but in https://github.com/rabbitmq/rabbitmq-website/issues/499#issuecomment-366483398 I mentioned what would accepted without scrutiny and long names wasn't on that list. Changes for the sake of changes is not something anyone is interested in.

Reply:
I think better names always improve code readability. This kind of name seems obvious to native speakers but for people who english it is not the mother tongue this kind of names can get trick really quick.

Like `x` is obvious that is for `exchange` because in the context there is an exchange and `x` sounds like `ex`. But for people who can read but not necessary speaker english `x` sounds like it would sound in their native tongue.

Sorry for not letting this clear the first time. If you still think it does not worth I can switch back.